src/sysinfo

```diff
@@ -196,6 +196,12 @@ function get_smartos_cpu_info()
         if [[ -z ${CPU_Version} ]]; then
             CPU_Version="Unknown"
         fi
+        # Platforms that include illumos#13560, smbios  may emit "sanitized"
+        # strings that are not valid json. So we need to pass these through
+        # JSON.stringify.
+        eval $(/usr/node/bin/node -e '
+            console.log("CPU_Version=" + JSON.stringify("'"$CPU_Version"'"))
+        ')
```

Comment:
This is the same fix as in 27dee23aaa6837651d8c1d1a376f308344100aca, it's just a different call to smbios.

There are no other calls to smbios where the value is used in the sysinfo output.